build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
True, installing `requirements_pip_extra.txt` would also work, it's just that it would also install `jupyter` and a bunch of other things necessary to build the docs but not to run the examples. In any case both would work.

Well yes, you could commit all your changes, and rebase on the current `FreeDiscovery/FreeDiscovery:master`, there would be some merge conflicts that would need to be resolved in the process...